<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <!-- event rendering -->
  <node name="dvs_renderer_left" pkg="dvs_renderer" type="dvs_renderer" ns="dvs_left" output="screen">
    <param name="display_method" value="pink-green"/>
  </node>
  
  <node name="dvs_renderer_right" pkg="dvs_renderer" type="dvs_renderer" ns="dvs_right" output="screen">
    <param name="display_method" value="pink-green"/>
  </node>
  
  <!-- rectificationa and stereo processing -->
  <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" ns="stereo" output="screen">
    <remap from="left/image_raw" to="/dvs_left/dvs_rendering"/>
    <remap from="left/camera_info" to="/dvs_left/camera_info"/>
    <remap from="right/image_raw" to="/dvs_right/dvs_rendering"/>
    <remap from="right/camera_info" to="/dvs_right/camera_info"/>
    
    <param name="approximate_sync" value="false"/>
  </node>

  <!-- display -->
  <node name="image_view_stereo_rect" pkg="image_view" type="stereo_view" output="screen">
  	<remap from="image" to="image_rect_color"/>
  	<param name="autosize" value="false"/>
  </node>
  
  <node name="image_view_stereo_unrect" pkg="image_view" type="stereo_view" output="screen">
  	<remap from="image" to="image_color"/>
  	<param name="autosize" value="false"/>
  </node>
    
</launch>
